rsocket-tck-drivers/src/main/java/io/rsocket/tckdrivers/client/JavaClientDriver.java

```diff
@@ -493,4 +526,35 @@ public void request(long n) {
       if (m > 0) pm.request(m);
     }
   }
+
+  /**
+   * A function that parses the file and extract the individual tests
+   *
+   * @param file The file to read as input.
+   * @return a list of individual tests. Each individual test is also a list of String.
+   */
+  public static List<List<String>> extractTests(File file) throws Exception {
```

Comment:
please refactor this to share code with the unit tests. Also can this return List<TckIndividualTest>?